.run/MultipleConfiguredPublicKeysSelectionLocationPropTest.run.xml

```diff
@@ -0,0 +1,37 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
What is the purpose of this file?

Reply:
I commited it by mistake... it's a run config I used for tests .... tnks for spotting